Library/ViewModels/ProjectPamphletViewModelTests.swift

```diff
@@ -840,30 +876,6 @@ final class ProjectPamphletViewModelTests: TestCase {
     }
   }
 
-  func testBackThisProjectButton_Tracking() {
```

Comment:
I removed this test because it wasn't really testing anything that we weren't already verifying in other tests 🤔